docs/content/blog/authelia-traefik-setup-guide/index.md

```diff
@@ -101,23 +129,24 @@ secrets:
 networks:
   proxy:
     external: true
-  authelia:
+  authelia: {}
 ```
 Notes:
 Timezone strings can be found [here](https://go.dev/src/time/zoneinfo_abbrs_windows.go)
 
 #### Basic Traefik Configuration
+Now we configure Traeifk.
 The following files contain the minimal Traefik configuration needed for Authelia integration:
 
-```yaml{title="traefik/config/traefik.yml"}
+```yaml{title="traefik/config/traefik.yaml"}
 # Base Traefik configuration
 api:
   dashboard: true
   debug: false
   insecure: false
 
 log:
-  level: INFO
+  level: DEBUG
```

Comment:
This is pretty noise from a default log level standpoint, I'd be inclined to suggest `INFO` or `ERROR` as the default.